### ISSUE TITLE
itsyouonline1.0: share information directly with other users, not just with organizations

### ISSUE THREAD

**Issue by robvanmieghem** [itsyouonline/identityserver#335]

**Comment by lucasvanhalst** [comment]
I guess this would just be used for API access to the info? As in I don't have to create some sort of 'profile' page

**Comment by robvanmieghem** [comment]
No, it is mostly not for api access but to make itysou.online more usable for end users.

**Comment by robvanmieghem** [comment]
And besides requesting information, it should also be possible to initiate the sharing yourself.

**Comment by lucasvanhalst** [comment]
OK, but then I need some kind of design for the 'profile' page.
